app/assets/javascripts/media_player_wrapper/mejs4_plugin_playlist_items.es6

```diff
@@ -79,6 +80,27 @@ Object.assign(MediaElementPlayer.prototype, {
    * @type {Object}
    */
   playlistItemsObj: {
+    /**
+     * Add click listeners forplaylist item links
+     * @function addRelatedItemListeners
+     * @return {void}
+     */
+    addRelatedItemListeners() {
+      // Handle click on entire RelatedItems area
+      // Filter only <a> element clicks; disregard all others
+      if (this.$relatedItems) {
+        this.$relatedItems.on('click', (e) => {
```

Comment:
Why do you need the click handler on the entire related items instead of attaching to the `<a>` tags themselves?

Reply:
@cjcolvar Sometimes this approach is more performant in event delegation, attaching one click handler to a parent element, rather than a bunch of click handlers to all child items.

Reply:
because the tags themselves aren't always visible and so can't be found to attach handlers.